chstrings/en.json

```diff
@@ -1,4 +1,5 @@
 {
+  "tooltitle": "Citation Hunt",
```

Comment:
I think it makes sense to mark this as optional on TranslateWiki, according to the definition of optional: https://translatewiki.net/wiki/FAQ#Optional_messages:_messages_rarely_need_to_be_translated

As far as I understand it, we don't have to do anything with the code though, so I'll just ask someone from TranslateWiki to do it after we merge this.